okio/src/jvmMain/kotlin/okio/HashingSink.kt

```diff
@@ -37,7 +37,7 @@ import javax.crypto.spec.SecretKeySpec
  * ByteString hash = hashingSink.hash();
  * ```
  */
-class HashingSink : ForwardingSink {
+actual class HashingSink : ForwardingSink, Sink { // Need to explicitly declare sink pending fix for https://youtrack.jetbrains.com/issue/KT-20641
```

Comment:
delete the docs from these classes? I don’t like having two copies that can get out of sync